src/plugins/content_management/opensearch_dashboards.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "contentManagement",
+  "version": "opensearchDashboards",
+  "server": false,
+  "ui": true,
+  "requiredPlugins": ["embeddable"],
```

Comment:
Guess requiredPlugins should be good enough? No need to declare it as requiredBundles.

Reply:
Seems it's needed as this plugin statically import modules from embeddable plugin

Reply:
Now I see the point, let me update this.